builtin/providers/aws/resource_aws_launch_configuration.go

```diff
@@ -89,27 +91,187 @@ func resourceAwsLaunchConfiguration() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+
+			"block_device": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+				Removed:  "Split out into three sub-types; see Changelog and Docs",
+			},
```

Comment:
No need to reproduce the deprecation warning here since we never supported it in the first place.